installer/scripts/genRubyInstaller.sh

```diff
@@ -28,16 +28,16 @@ substitute_arch()
     echo -n "$RESULT"
 }
 
-# This script will generate a "starting point" for an InstallBuilder data file.
-# For example, this generated ruby.data in the omsagent project. This is useful
-# for a package that has an install process (i.e. make install), but we need to
-# generate an install data file for it.
+# This script will generate a file that will contain the new
+# ruby  changes for ruby.data file based on SOURCE_DIR content to be included in ruby install builder 
+# since building ruby has an install process
+# and requires generating an install ruby data file for it.
 
-OUTPUT_RESULTS="/tmp/installBuilder-$USER"
+OUTPUT_RESULTS="/tmp/installBuilder-ruby-data-$USER"
 rm -f $OUTPUT_RESULTS
 
 # Evaluate any wildcards in the filename ...
-SOURCE_DIR=`stat -c "%n" ~/dev/work/oms/intermediate/*/ruby`
+SOURCE_DIR=`stat -c "%n" ../intermediate/*/100/ruby`
```

Comment:
Since SOURCE_DIR is using relative path from the build location, it will fail at line # 50 if the script is lunch not from the build location. But I agree that changing the echo to be more descriptive would be really useful. Changing it now.

# Verify validity of source directory

if [ -z "$SOURCE_DIR" -o ! -d $SOURCE_DIR ]; then
    echo "Source directory ($SOURCE_DIR) does not exist" >& 2
    exit 1
fi